cylc/flow/scripts/lint.py

```diff
@@ -162,6 +165,34 @@
 }
 
 
+def get_wallclock_directives():
+    """Get a set of directives equivalent to execution time limit"""
+    job_runner_manager = JobRunnerManager()
+    directives = {}
+    for module in pkgutil.iter_modules(job_runner_handlers.__path__):
+        directive = getattr(
+            job_runner_manager._get_sys(module.name),
+            'TIME_LIMIT_DIRECTIVE',
+            None
+        )
+        if directive:
+            directives[module.name] = directive
+    return directives
+
+
+WALLCLOCK_DIRECTIVES = get_wallclock_directives()
+
+
```

Comment:
Should this be check_wallclock_directives() now?

Reply:
Are you referring to next line? https://github.com/cylc/cylc-flow/blob/b42ff8b3c7bfc148a2c5d81d4e01833a431b276f/cylc/flow/scripts/lint.py#L186

Reply:
yea

Reply:
Legit point

Reply:
Have dealt with.